### ISSUE TITLE
Questions on training with less GPUs

### ISSUE THREAD

**Issue by cwchenwang** [cvlab-columbia/zero123#44]
Hi, authors thanks for your work. I'm training your repo with 2 A100 and set batch_size = 192, do I have to set accumulate_grad_batches to 4 since you trained on 8 A100s?

**Comment by ruoshiliu** [comment]
You should feel free to try anything :)  Batch_size = 192 x 8 is the setting that works for our experiments so you will need to do grad_acc = 4 to replicate our experiments. My guess is something smaller could still train without the loss diverging (e.g. 384 but with a smaller learning rate).

**Comment by Bhavay192** [comment]
Hi @cwchenwang @ruoshiliu I have just started using this repo and I'm working on main.py. Where can I set accumulate_grad_batches parameter? 
I am constantly facing the problem of Data loader worker exited unexpectedly as I set num_workers to any value and If I set it to 0, then it just stops randomly.
I'm using 2 GPUs of 48GB size NVIDIA A40.
